frontend-public/src/components/Forms/EditReportForm.js

```diff
@@ -21,8 +20,8 @@ export const EditReportForm = (props) => (
         <Form.Item label="Attached Files">
           <div>
             <p>
-              Note: you will be able to remove files up until the March 31, {moment().year()}{" "}
-              deadline. After that date, all attached files will be considered official submissions
+              Note: You will be able to remove files up until the end of the reports due date. 
```

Comment:
Are you trying specify the end of day on the date listed? Wording seems a little cluncky, but if UX/UI is fine with it, nothing to worry about.

Reply:
You've read it correct. The phrasing does make sense, and Aaron approved. Any ideas to improve?

Reply:
"You will be able to remove files up until the end of the day the report is due"?